applications/blackhole/src/blackhole_socket_callback.erl

```diff
@@ -39,6 +39,29 @@ recv(_SessionPid, _SessionId, {'event', _Ignore, <<"subscribe">>, SubscriptionJO
     end,
     {'ok', Context1};
 
+recv(SessionPid, SessionId, {'event', _Ignore, <<"unsubscribe">>, {[]}}, Context) ->
```

Comment:
Use the macro `?EMPTY_JSON_OBJECT` here instead of `{[]}`. You may have to add `-include_lib("whistle/src/wh_json.hrl").` at the top of the file.